.github/workflows/package-publish.yml

```diff
@@ -0,0 +1,67 @@
+name: Publish
+
+on:
+  pull_request:
```

Comment:
please note: that it is just for testing and in the release will be changed to push